text/0016-standard-modes.md

```diff
@@ -0,0 +1,292 @@
+
+The proposed initial set of modes is:
+```xml
+    <enum name="MAV_STANDARD_MODE">
+      <description>Standard modes with a well understood meaning across flight stacks and vehicle types.
+        For example, most flight stack have the concept of a "return" or "RTL" mode that takes a vehicle to safety, even though the precise mechanics of this mode may differ.
+        Modes may be set using MAV_CMD_DO_SET_STANDARD_MODE.
+      </description>
+      <entry value="0" name="MAV_STANDARD_MODE_NON_STANDARD">
+        <description>Non standard mode.
+          This may be used when reporting the mode if the current flight mode is not a standard mode.
+        </description>
+      </entry>
+      <entry value="1" name="MAV_STANDARD_MODE_POSITION_HOLD">
+        <description>Position mode (manual).
+          Position-controlled and stabilized manual mode.
+          When sticks are released vehicles return to their level-flight orientation.
+          Hovering vehicles actively brake and hold both position and altitude against wind and external forces,
+          Forward-traveling vehicles maintain current track and altitude.
+        </description>
+      </entry>
+      <entry value="2" name="MAV_STANDARD_MODE_ALTITUDE_HOLD">
+        <description>Altitude hold (manual).
+          Altitude-controlled and stabilized manual mode.
+          When sticks are released vehicles return to their level-flight orientation.
+          Hovering vehicles hold altitude but continue with existing momentum and may move with wind.
+          Forward-traveling vehicles maintain altitude but may be moved off their current heading by exernal forces.
+        </description>
+      </entry>
+      <entry value="3" name="MAV_STANDARD_MODE_RETURN_HOME">
+        <description>Return home mode (auto).
+          Automatic mode that returns vehicle to home via a safe flight path.
+          It may also automatically land the vehicle (i.e. RTL).
+          The precise flight path and landing behaviour depend on vehicle configuration and type.
+        </description>
+      </entry>
+      <entry value="4" name="MAV_STANDARD_MODE_SAFE_RECOVERY">
+        <description>Safe recovery mode (auto).
+          Automatic mode that takes vehicle to a predefined safe location via a safe flight path (rally point or mission defined landing) .
+          It may also automatically land the vehicle.
+          The precise return location, flight path, and landing behaviour depend on vehicle configuration and type.
+        </description>
+      </entry>
+      <entry value="5" name="MAV_STANDARD_MODE_MISSION">
+        <description>Mission mode (automatic).
+          Automatic mode that executes MAVLink missions.
+          Missions are executed from the current waypoint as soon as the mode is enabled.
+          If a mission cannot be executed the mission is paused.
```

Comment:
"If a mission cannot be executed the mission is paused."??  I think this is implementation specific, and it is somewhat unclear.  There are a number of viable actions when a mission can't be executed.  What does "mission is paused" actually mean?  What is the expected behaviour?  Suggest removing this line.

Reply:
Agreed thanks. Done.